modules/@apostrophecms/schema/index.js

```diff
@@ -1706,6 +1706,24 @@ module.exports = {
         });
       },
 
+      async getChoicesForQueryBuilder(field, query) {
+        const req = self.apos.task.getReq();
+        const allChoices = await self.getChoices(req, field);
```

Comment:
`self.getChoices` was already there and previously added in #4078, but never used in the builders